Updated mec172xevb_assy6906 documentation

wu-frances opened the pull request.

tejlmand reviewed (dismissed):
thanks for this contribution.

Some small comments regarding the build code.

MaureenHelm commented:
Thank you for your contribution!

This PR does a lot more than updating board documentation. Please edit the PR title and description to reflect that you're adding a new board. Please also review the failed compliance check details - there are many checkpatch errors that need to be addressed, and the commit messages need to have a body.

albertofloyd commented on boards/arm/mec172xmodular_assy6930/mec172xmodular_assy6930.dts:
```diff
@@ -0,0 +1,107 @@
+/* Initialize ECIA. Does not initialize child devices */
+&ecia {
+	status = "okay";
+};
```
@wu-frances this is missing several eSPI related entries that should be supported  so modular card can be used with host. i.e. girq24, espi0, kbc0, acpi_ec0

albertofloyd replied:
PR scope changed, comment not applicable

wu-frances committed:
mec172xevb_assy6906: update: documentation

Update mec172xevb_assy6906 documentation to include the correct
jumper settings. Fix documentation style issue. Clarify SPI image
generationprocess. Reduce size on large images.

Signed-off-by: Frances Wu <frances.wu@microchip.com>

albertofloyd commented:
@wu-frances looks good, I assume you will be raising separate PR for modular card.
Do squash the commits.

wu-frances commented:
Documentation for mec172xevb_assy6906 has been updated per comment.  The scope of the PR has been reduced to include just the MEC172x EVB documentation.  mec172xmodular_assy6930 board files and documentation will be submitted in a separate PR.

enjiamai reviewed (dismissed):
For mec172xevb_assy6906 setup, it has verified works.

tejlmand commented:
@wu-frances the commit message says:
> Original PR included mec172xmodular_assy6930 board files update.
Those files have been removed and will be included in separate
pull request.

This information is of no value once the PR is merged to main, so please remove those lines.

Extra info regarding commit messages can be seen here: https://docs.zephyrproject.org/latest/contribute/index.html#commit-message-body

MaureenHelm reviewed (changes requested):
Please reduce the size of the images, some are over 1 MB

tejlmand reviewed (approved):
lgtm - re-approved
